cmd/starter/c/starter.c

```diff
@@ -852,6 +853,28 @@ static void fix_streams(void) {
     }
 }
 
+static void fix_userns_devfuse_fd(struct starter *starter) {
```

Comment:
I think I understand why this is here to duplicate the fd - but could you write a short comment on why it's needed so it's easier to follow?

-edit- maybe not needed, I see a comment where it is called.